---
title: "01_create_consensus_peaks"
author: "Dakota Hunt"
date: "4/24/2023"
output: html_document
editor_options: 
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
options(stringsAsFactors = F)
library(tidyverse)
library(GenomicRanges)
source("../../util/my_class_functions.R")

# filepath to import peaks
basepath <- "/scratch/Shares/dahu3610"
peak_path <- "CLASS_2023/CLASSES/class_final_project/analysis/01_create_consensus_peaks"
broadpeakfilepath <- "/scratch/Shares/rinnclass/CLASS_2023/data/data/peaks"
```

# Loading in all .broadPeak files for all dbps and their replicates

```{r load .broadPeak files}

# using import peaks to import .broadPeak files (~10min)
peak_list <- import_peaks(consensus_file_path = broadpeakfilepath)

```

# Creating consensus peaks
```{r create consensus peaks}

# Create dbp object
dbp <- unique(sapply(names(peak_list), function(x) {
   unlist(strsplit(x, "_"))[1]
}))

# Create consensus_list object from dbp 
consensus_list <- lapply(dbp, consensus_from_reduced, peak_list)

# adding names to consensus_list
names(consensus_list) <- dbp

# saving consensus_list
save(consensus_list, file = "results/consensus_list.RData")

load("results/consensus_list.RData", verbose = T)
```

# Creating num_peaks_df
```{r num_peaks data frame}

# Create dataframe with dbp name and number of consensus peaks for that dbp
num_consensus_peaks <- sapply(consensus_list, length) %>% 
  as.data.frame() %>%
  rownames_to_column( var = "dbp") %>%
  dplyr::rename(number_consensus_peaks = ".")

# saving 
write_csv(num_consensus_peaks, "results/num_peaks_df.csv")

```


# Histogram of peak number for all dbps
```{r exploring number of peaks in consensus peaks}

# creating list of num_peaks per dbp
num_peaks <- sapply(consensus_list, length)
# plotting
hist(num_peaks, breaks = 1000)


```


# filtering consensus_list to dbps with > 1000 peaks
```{r filtered_consenus_list}

# filtering to 1000 peaks
filtered_consensus_list <- consensus_list[sapply(consensus_list, length) > 1000]

# saving 
save(filtered_consensus_list, file = "results/filtered_consensus_list.RData")
#load("results/filtered_consensus_list.RData", verbose = T)

# make object and csv for DBPs excluded for having 1000 or fewer peaks
excluded_dbps <- names(consensus_list[sapply(consensus_list, length) < 1000]) %>% as.data.frame()
# saving 
write.table(excluded_dbps, "results/excluded_dbps.csv")

```


# loading in genome features and creating annotation objects
```{r creating genome feature objects}

gencode_gr <- rtracklayer::import("/scratch/Shares/rinnclass/CLASS_2023/data/data/genomes/gencode.v32.annotation.gtf")

# gencode genes
gencode_genes <- gencode_gr[gencode_gr$type == "gene"] 

# mrna_genes
mrna_genes <- gencode_genes[gencode_genes$gene_type %in% "protein_coding"]

# lncrna_genes
lncrna_genes <- gencode_genes[gencode_genes$gene_type %in% "lncRNA"] 

# mrna_lncrna_genes
mrna_lncrna_genes <- gencode_genes[gencode_genes$gene_type %in% c("protein_coding","lncRNA")]

# lncrna_mrna_promoters
lncrna_mrna_promoters <- promoters(mrna_lncrna_genes, upstream = 1000, downstream = 1000)

# lncrna_gene_ids
lncrna_gene_ids <- mrna_lncrna_genes$gene_id[mrna_lncrna_genes$gene_type == "lncRNA"]

# mrna_gene_ids
mrna_gene_ids <-mrna_lncrna_genes$gene_id[mrna_lncrna_genes$gene_type == "protein_coding"]

```


# making num_peaks_df data frame of filtered_consensus_peak info
```{r creating num_peaks_df to track peak properties}

num_peaks_df <- data.frame("dbp" = names(filtered_consensus_list),
                           "num_peaks" = sapply(filtered_consensus_list, length))

# add total genome covered by peaks to num_peaks_df
num_peaks_df$total_peak_length <- sapply(filtered_consensus_list, function(x) sum(width(x)))

# sum rows to get total number of promoter overlaps
num_peaks_df$peaks_overlapping_promoters <- rowSums(promoter_peak_counts)

# lncrna promoter overlaps 
num_peaks_df$peaks_overlapping_lncrna_promoters <- rowSums(promoter_peak_counts[,lncrna_gene_ids])

# mrna promoter overlaps
num_peaks_df$peaks_overlapping_mrna_promoters <- rowSums(promoter_peak_counts[,mrna_gene_ids])

# Find overlaps with gene_bodies 
genebody_peak_counts <- count_peaks_per_feature(mrna_lncrna_genes, 
                                                filtered_consensus_list, 
                                                type = "counts")

# Count all gene body overlaps and add to num_peaks_df
num_peaks_df$peaks_overlapping_genebody <- rowSums(genebody_peak_counts)

# Count lncRNA gene body overlaps and add to num_peaks_df
num_peaks_df$peaks_overlapping_lncrna_genebody <- rowSums(genebody_peak_counts[,lncrna_gene_ids])

# Count mRNA gene body overlaps and add to num_peaks_df
num_peaks_df$peaks_overlapping_mrna_genebody <- rowSums(genebody_peak_counts[,mrna_gene_ids])


```

# adding TF type annotations for DBPs
```{r adding TF features to num_peaks_df}

# download TF annotations to results
url <- "https://www.cell.com/cms/10.1016/j.cell.2018.01.029/attachment/ede37821-fd6f-41b7-9a0e-9d5410855ae6/mmc2.xlsx"

destination_for_url <- "results/TF_annotations.xlsx"

# to download we can use download.file
download.file(url, destination_for_url)

#This didn't work so I just copied file from previous directory used in class.

# reading in TF annotations 
human_tfs <- readxl::read_excel("results/TF_annotations.xlsx",
                                sheet = 2, skip = 1)

# let's rename the 4th column to indicate if it is a TF.
names(human_tfs)[4] <- "is_tf"

# now let's intersect gene names that are in our ChIP data and has TF identity.
length(which(tolower(num_peaks_df$dbp) %in% tolower(human_tfs$Name)))
# 407 of the 430 have matching gene_names 

human_tfs <- human_tfs[tolower(human_tfs$Name) %in% tolower(num_peaks_df$dbp), 1:4]
# adding new column names
names(human_tfs) <- c("ensembl_id",
                      "dbp",
                      "dbd",
                      "tf")

# merging into num_peaks_df so that we get dbd and is.tf information in there
num_peaks_df <- merge(num_peaks_df, human_tfs, all.x = T)


```


# creating promoter peak occurence matrix
This makes a matrix where promoters are columns
Each will have a 1 if overlapped by a given dbp : 0 if no overlap

```{r promoter peak occurence matrix}

# running count_peaks_per_feature
promoter_peak_occurence <- count_peaks_per_feature(lncrna_mrna_promoters, filtered_consensus_list, 
                                               type = "occurrence")

# Let's double check that all lncrna & mrna genes are accounted for:
stopifnot(all(colnames(promoter_peak_occurence) == lncrna_mrna_promoters$gene_id))

# saving
write.table(promoter_peak_occurence, "results/lncrna_mrna_promoter_peak_occurence_matrix.tsv")

# Now let's use the 'data.frame()' fucntion. Set up a bunch of colnames and populate them.
peak_occurence_df <- data.frame("gene_id" = colnames(promoter_peak_occurence),
                                "gene_name" = lncrna_mrna_promoters$gene_name,
                                "gene_type" = lncrna_mrna_promoters$gene_type,
                                "chr" = lncrna_mrna_promoters@seqnames,   
                                "1kb_up_tss_start" = lncrna_mrna_promoters@ranges@start,
                                "strand" = lncrna_mrna_promoters@strand,
                                "number_of_dbp" = colSums(promoter_peak_occurence))

# saving
write_csv(peak_occurence_df, "results/peak_occurence_dataframe.csv")


```

# now make a promoter data_frame that tells which dbps are bound
```{r dbp centric promoter occurence}

# dbps on promoters object
DBPs_on_promoter <- lncrna_mrna_promoters %>%
                    as.data.frame() %>%
  dplyr::select(gene_id, gene_name)

# creating promoter dbps by pivot longer of promoter_peak_occurence_matrix
promoter_dbps <- promoter_peak_occurence %>%
  as.data.frame() %>%
  rownames_to_column("dbp") %>%
pivot_longer(2:ncol(.), names_to = "gene_id", values_to = "occurrence") %>%
  filter(occurrence == 1) %>%
  dplyr::select(-occurrence) %>%
  left_join(DBPs_on_promoter)

# checking Firre promoter
firre_promoter <- promoter_dbps %>%
  filter(gene_name == "FIRRE")

# XIST promoter (should be off since male cells)
XIST_promoter <- promoter_dbps %>%
  filter(gene_name == "XIST")

# SMYD1
SMYD1_promoter <- promoter_dbps %>%
  filter(gene_name == "SMYD1")

# saving
promoter_dbps_df <- promoter_dbps %>% as.data.frame()
write.csv(promoter_dbps, "results/promoter_dbps.csv")

```

# saving environment objects
```{r saving key objects in environment to load next session}

save(filtered_consensus_list, gencode_genes, lncrna_gene_ids, mrna_gene_ids, num_peaks_df, peak_occurence_df, promoter_peak_occurrence_matrix, lncrna_mrna_promoters, mrna_lncrna_genes, file = "results/peak_features.RData")


```





