---
title: "final_knit"
author: "Dakota Hunt"
date: "4/30/2023"
output: github_document
editor_options: 
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
options(stringsAsFactors = F)
library(tidyverse)
library(GenomicRanges)
library(ggplot2)
library(pheatmap)
source("../../util/my_class_functions.R")
source("../../util/_setup.R")
#source("../../util/intersect_functions.R")

# filepath to import peaks
basepath <- "/scratch/Shares/dahu3610"
peak_path <- "CLASS_2023/CLASSES/class_final_project/analysis/01_create_consensus_peaks"
broadpeakfilepath <- "/scratch/Shares/rinnclass/CLASS_2023/data/data/peaks"
```

# Loading in all .broadPeak files for all dbps and their replicates
```{r load broadPeak files}
# using import peaks to import broadPeak files (~10min)
peak_list <- import_peaks(consensus_file_path = broadpeakfilepath)
```

# Creating consensus peaks
```{r create consensus peaks}
# Create dbp object
dbp <- unique(sapply(names(peak_list), function(x) {
   unlist(strsplit(x, "_"))[1]
}))

# Create consensus_list object from dbp 
consensus_list <- lapply(dbp, consensus_from_reduced, peak_list)

# adding names to consensus_list
names(consensus_list) <- dbp

# saving consensus_list
save(consensus_list, file = "results/consensus_list.RData")

load("../01_consensus_peaks/results/consensus_list.RData", verbose = T)
```

# Creating num_peaks_df
```{r num_peaks data frame}

# Create dataframe with dbp name and number of consensus peaks for that dbp
num_consensus_peaks <- sapply(consensus_list, length) %>% 
  as.data.frame() %>%
  rownames_to_column( var = "dbp") %>%
  dplyr::rename(number_consensus_peaks = ".")

# saving 
write_csv(num_consensus_peaks, "results/num_peaks_df.csv")

```


# Histogram of peak number for all dbps
```{r exploring number of peaks in consensus peaks}

# creating list of num_peaks per dbp
num_peaks <- sapply(consensus_list, length)
# plotting
hist(num_peaks, breaks = 1000)


```


# filtering consensus_list to dbps with > 1000 peaks
```{r filtered_consenus_list}

# filtering to 1000 peaks
filtered_consensus_list <- consensus_list[sapply(consensus_list, length) > 1000]

# saving 
save(filtered_consensus_list, file = "results/filtered_consensus_list.RData")
#load("results/filtered_consensus_list.RData", verbose = T)

# make object and csv for DBPs excluded for having 1000 or fewer peaks
excluded_dbps <- names(consensus_list[sapply(consensus_list, length) < 1000]) %>% as.data.frame()
# saving 
write.table(excluded_dbps, "results/excluded_dbps.csv")

```


# loading in genome features and creating annotation objects
```{r creating genome feature objects}

gencode_gr <- rtracklayer::import("/scratch/Shares/rinnclass/CLASS_2023/data/data/genomes/gencode.v32.annotation.gtf")

# gencode genes
gencode_genes <- gencode_gr[gencode_gr$type == "gene"] 

# mrna_genes
mrna_genes <- gencode_genes[gencode_genes$gene_type %in% "protein_coding"]

# lncrna_genes
lncrna_genes <- gencode_genes[gencode_genes$gene_type %in% "lncRNA"] 

# mrna_lncrna_genes
mrna_lncrna_genes <- gencode_genes[gencode_genes$gene_type %in% c("protein_coding","lncRNA")]

# lncrna_mrna_promoters
lncrna_mrna_promoters <- promoters(mrna_lncrna_genes, upstream = 1000, downstream = 1000)

# lncrna_gene_ids
lncrna_gene_ids <- mrna_lncrna_genes$gene_id[mrna_lncrna_genes$gene_type == "lncRNA"]

# mrna_gene_ids
mrna_gene_ids <-mrna_lncrna_genes$gene_id[mrna_lncrna_genes$gene_type == "protein_coding"]

```


# making num_peaks_df data frame of filtered_consensus_peak info
```{r creating num_peaks_df to track peak properties}

num_peaks_df <- data.frame("dbp" = names(filtered_consensus_list),
                           "num_peaks" = sapply(filtered_consensus_list, length))

# counting promoter overlaps
promoter_peak_counts <- count_peaks_per_feature(lncrna_mrna_promoters, filtered_consensus_list, type = "counts")

# add total genome covered by peaks to num_peaks_df
num_peaks_df$total_peak_length <- sapply(filtered_consensus_list, function(x) sum(width(x)))

# sum rows to get total number of promoter overlaps
num_peaks_df$peaks_overlapping_promoters <- rowSums(promoter_peak_counts)

# lncrna promoter overlaps 
num_peaks_df$peaks_overlapping_lncrna_promoters <- rowSums(promoter_peak_counts[,lncrna_gene_ids])

# mrna promoter overlaps
num_peaks_df$peaks_overlapping_mrna_promoters <- rowSums(promoter_peak_counts[,mrna_gene_ids])

# Find overlaps with gene_bodies 
genebody_peak_counts <- count_peaks_per_feature(mrna_lncrna_genes, 
                                                filtered_consensus_list, 
                                                type = "counts")

# Count all gene body overlaps and add to num_peaks_df
num_peaks_df$peaks_overlapping_genebody <- rowSums(genebody_peak_counts)

# Count lncRNA gene body overlaps and add to num_peaks_df
num_peaks_df$peaks_overlapping_lncrna_genebody <- rowSums(genebody_peak_counts[,lncrna_gene_ids])

# Count mRNA gene body overlaps and add to num_peaks_df
num_peaks_df$peaks_overlapping_mrna_genebody <- rowSums(genebody_peak_counts[,mrna_gene_ids])


```

# adding TF type annotations for DBPs
```{r adding TF features to num_peaks_df}

#This didn't work so I just copied file from previous directory used in class.

# reading in TF annotations 
human_tfs <- readxl::read_excel("results/TF_annotations.xlsx",
                                sheet = 2, skip = 1)

# let's rename the 4th column to indicate if it is a TF.
names(human_tfs)[4] <- "is_tf"

# now let's intersect gene names that are in our ChIP data and has TF identity.
length(which(tolower(num_peaks_df$dbp) %in% tolower(human_tfs$Name)))
# 407 of the 430 have matching gene_names 

human_tfs <- human_tfs[tolower(human_tfs$Name) %in% tolower(num_peaks_df$dbp), 1:4]
# adding new column names
names(human_tfs) <- c("ensembl_id",
                      "dbp",
                      "dbd",
                      "tf")

# merging into num_peaks_df so that we get dbd and is.tf information in there
num_peaks_df <- merge(num_peaks_df, human_tfs, all.x = T)


```


# creating promoter peak occurence matrix
This makes a matrix where promoters are columns
Each will have a 1 if overlapped by a given dbp and a 0 if no overlap

```{r peak occurence df}

# running count_peaks_per_feature
promoter_peak_occurence <- count_peaks_per_feature(lncrna_mrna_promoters, filtered_consensus_list, 
                                               type = "occurrence")

# Let's double check that all lncrna & mrna genes are accounted for:
stopifnot(all(colnames(promoter_peak_occurence) == lncrna_mrna_promoters$gene_id))

# saving
write.table(promoter_peak_occurence, "results/promoter_peak_occurence_matrix.tsv")

# Now let's use the 'data.frame()' fucntion. Set up a bunch of colnames and populate them.
peak_occurence_df <- data.frame("gene_id" = colnames(promoter_peak_occurence),
                                "gene_name" = lncrna_mrna_promoters$gene_name,
                                "gene_type" = lncrna_mrna_promoters$gene_type,
                                "chr" = lncrna_mrna_promoters@seqnames,   
                                "1kb_up_tss_start" = lncrna_mrna_promoters@ranges@start,
                                "strand" = lncrna_mrna_promoters@strand,
                                "number_of_dbp" = colSums(promoter_peak_occurence))

# saving
write_csv(peak_occurence_df, "results/peak_occurence_dataframe.csv")


```

# saving important environment objects
```{r saving important environment objects so they can be loaded in future}

save(filtered_consensus_list, gencode_genes, lncrna_gene_ids, mrna_gene_ids, num_peaks_df, peak_occurence_df, promoter_peak_occurence, lncrna_mrna_promoters, mrna_lncrna_genes, file = "results/peak_features.RData")


```


#####################################

# loading in environment objects previously created
```{r load env objects}

load("results/peak_features.RData", verbose = T)

```


# Peaks per dbp histogram
```{r plotting peaks per dbp}

 ggplot(num_peaks_df, aes(x = num_peaks)) + 
  geom_histogram(bins = 100)

# Save figure
ggsave("figures/num_peaks_hist.pdf")

```


# Plot genome coverage (total peak length) against the number of peaks for each DBP
```{r peaks vs coverage}

ggplot(num_peaks_df, aes(x = num_peaks, y = total_peak_length)) +
  geom_point() + 
  ylab("BP covered") +
  xlab("Number of peaks") +
  ggtitle("Peak count vs. Genome Coverage")

# Save figure
ggsave("figures/peak_num_vs_coverage.pdf")


```
Result: there is an approximately linear relationship between the number of peaks and total coverage, which is to be expected.

# What is the distribution of promoter overlaps versus gene-bodies?
```{r promoter overlaps vs gene bodies}
# Histogram for promoters
 ggplot(num_peaks_df, aes(x = peaks_overlapping_promoters)) + 
  geom_histogram(bins =25)

# Histogram for gene bodies
 ggplot(num_peaks_df, aes(x = peaks_overlapping_genebody)) + 
  geom_histogram(bins =25)
 
 # Double histogram
 
ggplot(num_peaks_df) +
  labs(fill="Variable name",x="Peaks overlapping feature",y="Count (# of DBPs)")+
  geom_histogram(aes(x=peaks_overlapping_genebody, fill= "Gene bodies",  color = "blue"), alpha = 0.2 ) + 
  geom_histogram(aes(x=peaks_overlapping_lncrna_promoters, fill= "Promoters", color = "red"), alpha = 0.2) +
  scale_fill_manual(values = c("blue","red"))+
  scale_colour_manual(values = c("blue","red"), guide=FALSE)
ggsave("figures/promoter_vs_genebody_overlap.pdf")
    

                 
```


# plotting num peaks on promoters vs number of peaks total for each DBP

```{r number of DBPS on promoters}
# Plotting x as num_peaks and y  as the number overlapping promoters.

ggplot(num_peaks_df,
       aes(x = num_peaks, y = peaks_overlapping_promoters)) +
  geom_point() +
  
  geom_abline(slope = 1, linetype="dashed") +
  geom_smooth(method = "lm", se=F, formula = 'y ~ x',
              color = "#a8404c") +
  ylim(0,60100) +
  xlim(0,60100) +

  # adding labels
xlab("Peaks per DBP") +
  ylab("Number of peaks overlapping promoters") +
  ggtitle("Relationship Between Number of DBP Peaks and Promoter Overlaps")

ggsave("figures/peak_num_vs_promoter_coverage.pdf")

```
Result: there appears to be a saturation point at which there is no longer an increase in the number of peaks overlapping promoters even as total peak number for the DBP increases.

# plotting num peaks over gene bodies vs number of peaks total for each DBP
```{r peak coverage on gene bodies}

ggplot(num_peaks_df,
       aes(x = num_peaks, y = peaks_overlapping_genebody)) +
  xlab("Peaks per DBP") +
  ylab("Number of peaks overlapping genes") +
  ggtitle("Relationship Between Number of DBP Peaks and Gene Body Overlaps")+
  geom_point() +
  geom_abline(slope = 1, linetype="dashed") +
  geom_smooth(method = "lm", se=F, formula = 'y ~ x',
              color = "#a8404c") +
  ylim(0,60100) +
  xlim(0,60100)

# saving
ggsave("figures/peak_num_vs_gene_body_coverage.pdf")
```
Result: Gene bodies make up almost all the places of binding in the genome. There is almost a 1:1 correlation between total peak number and peaks overlapping promoters for these DBPs.

# Density plot of binding events
Making a density plot of number of DBPs bound per promoter

```{r density plot of DBP localization events}

ggplot(peak_occurence_df, aes(x = number_of_dbp)) +
geom_density(alpha = 0.2, color = "#424242", fill = "#424242") +
  xlab(expression("Number of DBPs")) +
  ylab(expression("Density")) +
  ggtitle("Promoter binding events",
          subtitle = "mRNA and lncRNA genes") 

# saving
ggsave("figures/num_binding_events_per_promoter.pdf")

# Colored by gene_type
# Density plot
ggplot(peak_occurence_df, aes(x = number_of_dbp, fill = gene_type)) +
geom_density(alpha = 0.2) +
  xlab(expression("Number of DBPs")) +
  ylab(expression("Density")) +
  ggtitle("Promoter binding events",
          subtitle = "mRNA and lncRNA genes") 
# saving
ggsave("figures/num_binding_events_per_promoter_by_genetype.pdf")


```
Result: promoter binding is bimodal. Most promoters have less than 50 DBPs but there is a group that has more than 200 DBPs bound! It is clear that mRNA genes are more likely appear in this superbinding group than lncRNA genes.

# promoters without binding events
Finding promoters that don't have any DBPs bound
```{r promoters with out binding events}

unbound_promoters <- peak_occurence_df %>% 
  filter(peak_occurence_df$number_of_dbp < 1)

# how many unbound promoters are there?
nrow(unbound_promoters)
# so there are 9448 promoters that don't have any DBPs bound
write_csv(unbound_promoters, "results/unbound_promoters.csv")

```

# lncRNA versus mRNA promoter binding
Comparing the binding patterns of lncRNA vs mRNA promoters.
```{r lncrna vs mrna promoter binding}


ggplot(num_peaks_df, aes(x = num_peaks)) +
  geom_point(aes(y = peaks_overlapping_lncrna_promoters), color = "red") +
  geom_point(aes(y = peaks_overlapping_mrna_promoters), color = "black") +
  geom_smooth(aes(y = peaks_overlapping_lncrna_promoters), method = "lm", se = FALSE, formula = "y ~ x") +
  geom_smooth(aes(y = peaks_overlapping_mrna_promoters), method = "lm", se = FALSE, formula = "y ~ x")


# saving
ggsave("figures/lncRNA-mRNA-promoter_binding.pdf")

# lncRNA in red, mRNA in black



```

# How many of these 430 DBPs are TFs? What is the most represented type of DBD?
```{r DBP type}
#How many proteins are TFs?
peak_TF_table <- table(num_peaks_df$tf, useNA = "always")
peak_TF_table

#What is the most represented type of DBD?
peak_DBD_table <- table(num_peaks_df$dbd, useNA = "always")
peak_DBD_table


```
Result: 344 proteins are TFs, 63 are not, and 23 have a value of NA.
Result: C2H2 zinc fingers are the most common type of DNA-binding domain, representing 169 of the 430 DBPs.


###################################
# SUPERBINDER ANALYSIS

# Create superbinder_promoters_df object and table of super-binding promoters
```{r superbinder promoters}

#Defining superbinding promoters as those with over 200 DBPs bound
superbinder_promoters_df <- peak_occurence_df %>% 
  filter(peak_occurence_df$number_of_dbp > 200)

# how many are there?
nrow(superbinder_promoters_df)
# so there are 11,689 promoters that have over 200 binding events

#  let's put it in a folder called results. We will always use this folder structure
write_csv(superbinder_promoters_df, "results/superbinder_promoters.csv")
```

# Create normal_promoters object and table of normal promoters
```{r normal promoters}
# Defining normal promoters as those with between 1 and 200 DBPs bound. 
# Promoters with zero DBPS bound were previously classified as unbound promoters. 
normal_promoters_df <- peak_occurence_df %>% 
  filter(peak_occurence_df$number_of_dbp <= 200 & peak_occurence_df$number_of_dbp >= 1)

# how many are there?
nrow(normal_promoters_df)
# so there are 15677 promoters that have between 1 to 200 binding events.

#  let's put it in a folder called results. We will always use this folder structure
write_csv(normal_promoters_df, "results/normal_promoters.csv")
```
Result: 9448 unbound promoters, 15677 normal promoters, 11689 superbinder promoters.

# Gene ontology analysis via Enrichr
```{r GO analysis via Enricher}
# 1. Download superbinder_promoters.csv via Cyberduck 
# 2. Open in Excel
# 3. Copy list of gene names and paste into Enrichr (https://maayanlab.cloud/Enrichr)
```
Results:
Superbinder promoters:

GO Biological Processes (sorted by p-value)
1. Ribosome biogenesis
2. rRNA processing
3. mRNA processing
4. Gene expression 
5. Nuclear-transcribed mRNA catabolic process

GO Molecular Function (sorted by p-value)
1. RNA binding
2. rRNA binding
3. Endoribonuclease activity
4. mRNA binding
5. DNA replication origin binding

This is really interesting as it seems as if a lot of genes associated with regulation of gene expression have lots of DBPs bound at there promotes and thus likely undergo complex regulation themselves.


# mRNA vs lncRNA superbinder promoters
```{r mRNA vs lncRNA superbinders}
# Look at how many superbinders are lncRNA vs mRNA genes
superbinder_type <- table(superbinder_promoters_df$gene_type)
superbinder_type

#Create new obejcts containing just mRNA and lncRNA superbinders
mRNA_superbinder_promoters <- superbinder_promoters_df %>% 
  filter(superbinder_promoters_df$gene_type == 'protein_coding')

lncRNA_superbinder_promoters <- superbinder_promoters_df %>% 
  filter(superbinder_promoters_df$gene_type == 'lncRNA')

```
Result: I found 9177 mRNA superbinder promoters and 2512 lncRNA superbinder promoters. This suggest the majority of superbinders are mRNA (protein-coding) genes.



########################################
# loading in promoter_peak_occurence_matrix
```{r promoter peak occurence matrix}

# Read in promoter peak occurence matrix 
promoter_peak_occurence_matrix <- read.table("results/promoter_peak_occurence_matrix.tsv")

# Converting to a matrix format for correlation analysis
promoter_peak_occurence_matrix <- as.matrix(promoter_peak_occurence_matrix)
# This is a large matrix with a value of 1 if a DBP is bound to a promoter and 0 if it is not.


```

# creating distance matrix & dendrogram
```{r distance matrix and dendrogram}

# creating distance matrix
peak_occurence_dist <- dist(promoter_peak_occurence_matrix, method = "binary")

# clustering distance values
bin_hier <- hclust(peak_occurence_dist, method = "complete")

# This is a lot of ggplot -- on purpose
# please read through what is happening and parameter usage

 ggdendro::ggdendrogram(bin_hier, rotate = FALSE,  size = 3, 
                       theme_dendro = TRUE) +
   # 90 degree rotation to right
   coord_flip() +
   scale_y_continuous() +
   # adds label
   scale_x_continuous(position = "top") +
   # subsection to labels in order of clustering
   # ? seq_along
   scale_x_continuous(breaks = seq_along(bin_hier$labels[bin_hier$order]),
                      
                      # adding labels that are in the order 'column'
             labels = bin_hier$labels[bin_hier$order], position = "top",
             expand = c(0,0)) +
   theme(axis.text.x = element_text(angle = 90, hjust  = 1)) + 
   theme(axis.text.y = element_text(angle = 0,hjust = 1)) +
   scale_y_reverse(expand = c(0.01, 0)) +
   theme(
     plot.background = element_blank(),
     panel.grid.major = element_blank(),
   panel.grid.minor = element_blank(),
     panel.border = element_blank()
   )

ggsave("figures/ggdendro_plot.pdf", height = 50, width = 12, limitsize = F)


```
Result: One interesting example is that ATF5 clusters closely with ZNF34 and ZNF343. ATF5 is highly abundant in the liver and is a member of the ATF/cAMP response element-binding protein family, which has been associated with differentiation, proliferation, and survival in several tissues and cell types. ATF5 is upregulated in response to various stress conditions, suggesting these two zinc fingers may also play a role in stress response in the liver by working with ATF5. ATF5 is also thought to stimulate differentiation of liver cells, and so these ZNFs may be involved in hepatocyte differentiation.

# Clustering of lncRNA and mRNA seperately


# lncRNA promoters ggdendro
```{r lncRNA promoter clustering}
# Create annotation for just lncRNA promoters
lncrna_promoters <- lncrna_mrna_promoters[lncrna_mrna_promoters$gene_type %in% "lncRNA"] 

# Now use indexing to separate peak_occurrence_matrix into just lncRNA and mRNA
lncrna_peak_occurence <- promoter_peak_occurence_matrix[,lncrna_promoters$gene_id]

# Clustering
bin_hier_lncrna <- hclust(dist(lncrna_peak_occurence, method = "binary"))

# Now plotting with ggdendro
ggdendro::ggdendrogram(bin_hier_lncrna, rotate = T,  size = 3)
 
# Saving
ggsave("figures/lncrna_hclust_binary_dist.pdf", height = 49, width = 6)


```

# mRNA promter ggdendro
```{R mRNA promoter clustering}
# Create annotation for just mRNA promoters
mrna_promoters <- lncrna_mrna_promoters[lncrna_mrna_promoters$gene_type %in% "protein_coding"] 

# Create peak occurence matrix for just mRNA promoters
mrna_peak_occurence <- promoter_peak_occurence_matrix[,mrna_promoters$gene_id]

# getting the distance matrix for only mRNA promoters  
bin_hier_mrna <- hclust(dist(mrna_peak_occurence, method = "binary"))
 
# plotting with ggdendro
ggdendro::ggdendrogram(bin_hier, rotate = TRUE,  size = 3)

# saving
ggsave("figures/mrna_hclust_binary_dist.pdf", height = 44, width = 6)

```
Results: ATF5 still clusters with ZNF34 and ZNF343 even when we cluster just by lncRNA or mRNA promoters. Overall clustering appears to be generally the same with some minor differences. There may be significant differences that I am missing due to size of dendrogram.


# Using profile_tss for all 430 DBPs
```{r metaplot DF of binding profiles over promoter}

# establishing DF
metaplot_df <- data.frame(x = integer(), dens = numeric(), dbp = character())

# for loop to populate DF 
for(i in 1:length(filtered_consensus_list)) {
  print(names(filtered_consensus_list)[[i]])
  tmp_df <- profile_tss(filtered_consensus_list[[i]], lncrna_mrna_promoters)
  tmp_df$dbp <- names(filtered_consensus_list)[[i]]
  metaplot_df <- bind_rows(metaplot_df, tmp_df)
  
}

# saving
write_rds(metaplot_df, "results/metaplot_df_final.rds")

```
  

# creating distance matrix of binding profile correlations
```{r scaling and plotting dendrogram of binding similarity by promoter}

metaplot_filtered_matrix <- metaplot_df %>% 
  pivot_wider(names_from = x, values_from = dens) %>%
  column_to_rownames("dbp") %>%
  as.matrix()
mm_scaled <- t(scale(t(metaplot_filtered_matrix)))
metaplot_hclust <- hclust(dist(mm_scaled), method = "complete")

# plotting relationship between binding profiles
plot(metaplot_hclust)
pdf("figures/tss_profile_dendrogram.pdf", height = 10, width = 27)
par(cex=0.3)
plot(metaplot_hclust)
dev.off()
```



# establishing lncRNA and mRNA promoters (+/- 1kb)
```{r create lncRNA and mRNA promoters }

# creating promoters just in case:
lncrna_promoters <- lncrna_mrna_promoters[lncrna_mrna_promoters$gene_type == "lncRNA"]
mrna_promoters <- lncrna_mrna_promoters[lncrna_mrna_promoters$gene_type == "protein_coding"]


```

# metaplots for each DBP by lncRNA and mRNA promoters
```{r}

#setting up lncrna DF.
lncrna_metaplot_df <- data.frame(x = integer(), dens = numeric(), dbp = character())

# for loop to populate DF with overlap density in lncrna promoters
for(i in 1:length(filtered_consensus_list)) {
  print(names(filtered_consensus_list)[[i]])
  tmp_df <- profile_tss(filtered_consensus_list[[i]], lncrna_mrna_promoters = lncrna_promoters)
  tmp_df$dbp <- names(filtered_consensus_list)[[i]]
  lncrna_metaplot_df <- bind_rows(lncrna_metaplot_df, tmp_df)
  
}

# saving
write_rds(lncrna_metaplot_df, "results/lncRNA_metaplot_df_final.rds")

# now for mRNAs 
mrna_metaplot_df <- data.frame(x = integer(), dens = numeric(), dbp = character())

# for loop to populate mRNA_metaplot
for(i in 1:length(filtered_consensus_list)) {
  print(names(filtered_consensus_list)[[i]])
  tmp_df <- profile_tss(filtered_consensus_list[[i]], lncrna_mrna_promoters = mrna_promoters)
  tmp_df$dbp <- names(filtered_consensus_list)[[i]]
  mrna_metaplot_df <- bind_rows(mrna_metaplot_df, tmp_df)
  
}

# saving mRNA metaplots
write_rds(mrna_metaplot_df, "results/mrna_metaplot_df_final.rds")


# now adding the information of gene type
mrna_metaplot_df$gene_type <- "mRNA"
lncrna_metaplot_df$gene_type <- "lncRNA"
combined_metaplot_profile <- bind_rows(mrna_metaplot_df, lncrna_metaplot_df)

# saving
write_rds(mrna_metaplot_df, "results/metaplot_df_final.rds")


# pdf(file = "figures/mega_plot_test.pdf")
ggplot(combined_metaplot_profile, 
       aes(x = x, y = dens, color = gene_type )) +
  geom_vline(xintercept = 0, lty = 2) + 
  geom_line(size = 1.5) + 
  facet_wrap(dbp ~ ., scales = "free_y") +
  ggtitle("Promoter Metaplot") + 
  scale_x_continuous(breaks = c(-1000, 0, 1000),
                     labels = c("-1kb", "TSS", "+1kb"),
                     name = "") + 
  ylab("Peak frequency") +
 scale_color_manual(values = c("#424242","#a8404c"))

# saving
ggsave("figures/mega_meta_plot_lncRNA-mRNA.pdf", width = 49, height = 12)
# ggsave("figures/mega_meta_plot_lncRNA-mRNA.pdf", limitsize = F)


```
Results: Generally, most DBPs have the same pattern for both mRNA and lncRNA promoters. Some proteins appear to be more enriched at the TSS of mRNAs such as RCOR1, FOXM1, etc. 
Interestingly, H3K4me1, H3K79me2, H4K20me1 are noticeably depleted at the TSS on mRNA genes compared to lncRNA genes. 
EZH2 has an interesting pattern where it appears to peak at lncRNA TSS's but is noticeably depleted at the TSS of mRNAs, with two peaks up and downstream of the TSS.


# Make a metaplot of DBPS only on Super-binders versus regular promoters
```{r}
superbinder_promoters <- lncrna_mrna_promoters[lncrna_mrna_promoters$gene_id %in% superbinder_promoters_df$gene_id] 

normal_promoters <- lncrna_mrna_promoters[lncrna_mrna_promoters$gene_id %in% normal_promoters_df$gene_id] 

#setting up superbinder metaplot DF.
superbinder_metaplot_df <- data.frame(x = integer(), dens = numeric(), dbp = character())

for(i in 1:length(filtered_consensus_list)) {
  print(names(filtered_consensus_list)[[i]])
  tmp_df <- profile_tss(filtered_consensus_list[[i]], lncrna_mrna_promoters = superbinder_promoters)
  tmp_df$dbp <- names(filtered_consensus_list)[[i]]
  superbinder_metaplot_df <- bind_rows(superbinder_metaplot_df, tmp_df)
  
}

# saving
write_rds(superbinder_metaplot_df, "results/superbinder_metaplot_df_final.rds")

# now for normal promoters 
normal_metaplot_df <- data.frame(x = integer(), dens = numeric(), dbp = character())

for(i in 1:length(filtered_consensus_list)) {
  print(names(filtered_consensus_list)[[i]])
  tmp_df <- profile_tss(filtered_consensus_list[[i]], lncrna_mrna_promoters = normal_promoters)
  tmp_df$dbp <- names(filtered_consensus_list)[[i]]
  normal_metaplot_df <- bind_rows(normal_metaplot_df, tmp_df)
  
}

# saving mRNA metaplots
write_rds(normal_metaplot_df, "results/normal_metaplot_df_final.rds")


# now adding the information of gene type
superbinder_metaplot_df$gene_type <- "superbinder"
normal_metaplot_df$gene_type <- "normal"
combined_super_normal_metaplot_profile <- bind_rows(superbinder_metaplot_df, normal_metaplot_df)

# saving
write_rds(combined_super_normal_metaplot_profile, "results/superbinder_metaplot_df_final.rds")


# plotting
ggplot(combined_super_normal_metaplot_profile, 
       aes(x = x, y = dens, color = gene_type )) +
  geom_vline(xintercept = 0, lty = 2) + 
  geom_line(size = 1.5) + 
  facet_wrap(dbp ~ ., scales = "free_y") +
  ggtitle("Promoter Metaplot") + 
  scale_x_continuous(breaks = c(-1000, 0, 1000),
                     labels = c("-1kb", "TSS", "+1kb"),
                     name = "") + 
  ylab("Peak frequency") +
 scale_color_manual(values = c("#424242","#a8404c"))

# saving
ggsave("figures/mega_meta_plot_super-normal.pdf", width = 49, height = 12)
```
Results: Generally, most DBPs have similar binding patterns at normal and superbinder promoters. NFE2L appears to be more enriched at the TSS of superbinder promoters compared to normal promoters. H3K4me1 and H3K79me2 and H4K20me1 appear to be depleted at the TSS of superbinder promoters compared to normal promoters. ZNF460 shows a really interesting pattern and appears to be more abundant on superbinder promoters vs normal promoters.

######################################
Binding vs Expression

# load in chipseq data from analysis/01_peak_feautres

```{r loading in chipseq data}
load("results/peak_features.RData", verbose = T)
```

# Reading in sample sheet

```{r read in sample_sheet}
# First let's read in the sample sheet to know what is what
samplesheet <- read_rds("../../../05_R_analyses/05_RNAseq/01_differential_expression/results/final_samplesheet.rds")
```

# reading in TPM values from Salmon for analyses

```{r reading in salmon Salmon TPMs}

# reading in salmon tpm
salmon_tpm <- read.csv("../../../05_R_analyses/05_RNAseq/00_RNAseq_download_NF_core_pipeline/00_NF_CORE_RNAseq_Pipeline_run/results/salmon/salmon_merged_gene_tpm.csv")

# TPM table is in same order as samplesheet
tpm <- salmon_tpm %>% 
  pivot_longer(cols = 2:ncol(.), names_to = "sample_id", values_to = "tpm") %>%
  merge(samplesheet) %>%
  group_by(gene_id, condition) %>%
  summarize(tpm = mean(tpm, na.rm = T)) %>%
  pivot_wider(names_from = condition, values_from = tpm, names_prefix = "tpm_")
```



# Plotting binding versus expression
# reading in promoter peak features dataframe

```{r loading in peak features data frame }

# peak_occurence_df is loaded in  environment and contains how many DBPs bind to each promoter (rows)
peak_occurence_df <- read_csv("results/peak_occurence_dataframe.csv")

# Merge TPM data with this dataframe
promoter_features_df <- merge(peak_occurence_df, tpm)

# saving
write.csv(promoter_features_df, "results/promoter_feature_df_tpm.csv")


```

# Plot DBP binding vs total RNA expression
```{r DBP promoter binding versus total RNA expression}

# plotting binding vs total RNA expression
ggplot(promoter_features_df, 
            aes(y = log2(tpm_homo_sapiens_hepg2 + 0.001), x = number_of_dbp, color = gene_type)) + 
geom_point(data = promoter_features_df %>% filter(tpm_homo_sapiens_hepg2 > 0.001),
             shape = 17, alpha = 0.7) +
  geom_smooth(method = 'gam', formula = y ~ s(x, bs = "cs")) +
  geom_smooth(method = "lm") +
  
  
  scale_x_continuous(expand = c(0,0)) +
  scale_color_manual(values = c("#a8404c", "#424242"), name = "Gene type") + 
  ggtitle("Expression vs. promoter binding events") + 
  xlab(expression('Number of TFs')) +
  ylab(expression(log[2](TPM))) 

ggsave("figures/binding_vs_expression_total_rna.pdf")
```
Results: There is a linear trend with number of DBPS and expression levels being positively correlated. There is a population of genes that are superbinding yet have low expression (reservoirs?)


# Abundance of genes in each cellular fraction
```{r TPM of genes in each fraction}
# First we need to the tpm DF into a matrix

tpm_matrix <- tpm %>% 
  column_to_rownames("gene_id") %>%
  as.matrix()
tpm_scaled <- t(scale(t(tpm_matrix)))
tpm_scaled <- tpm_scaled[complete.cases(tpm_scaled),]


# plotting
new.env()
pdf("figures/heatmap_expression.pdf", height =49, width = 12)
pheatmap::pheatmap(tpm_scaled, show_rownames = FALSE)
graphics.off()

```
RESULT: Most RNAs are abundant in the nuclear fraction and interestingly there are some RNAs expressed in the total samples that are not present in other fractions.


# Making a heatmap of nuclear versus cytoplasmic expression
```{r TPM of genes in nuc vs cyto fractions}
# Filter tpm_scaled
tpm_scaled_nuc_cyto <- tpm_scaled[,colnames(tpm_scaled) == "tpm_homo_sapiens_cytosolic_fraction"
                                  | colnames(tpm_scaled) == "tpm_homo_sapiens_nuclear_fraction"]
# plotting
new.env()
pdf("figures/nuc_cyto_heatmap_expression.pdf", height =49, width = 12)
pheatmap::pheatmap(tpm_scaled_nuc_cyto, show_rownames = FALSE)
graphics.off()

```


# Differential expression analysis
```{r read in deseq_results_df and analyze}
# Comparing nuclear vs cytoplasmic expression. Read in results dataframe from DESeq2 (done during class)
deseq_res_shrunken_df <- read_rds("../../../05_R_analyses/05_RNAseq/01_differential_expression/results/nuclear_cyto_results_df.rds")

#Define which genes are nuclear
nuclear_genes <- deseq_res_shrunken_df%>% 
  filter(log2FoldChange < -1, padj < 0.05) %>%
  as.data.frame()

# saving file
write_csv(nuclear_genes, "results/nuclear_genes.csv")

# Cytoplasmic genes:
cyto_genes <- deseq_res_shrunken_df %>%
  filter(log2FoldChange > 1, padj < 0.05)

# saving file
write_csv(cyto_genes, "results/cyto_genes.csv")

# How many total genes are significantly enriched in nucleus? 5978
nrow(nuclear_genes)

# Breakdown of lncRNA vs mRNA genes in nucleus
nuclear_lncRNA_genes <- nuclear_genes[nuclear_genes$gene_id %in% lncrna_gene_ids,]
nuclear_mRNA_genes <- nuclear_genes[nuclear_genes$gene_id %in% mrna_gene_ids,]
nuclear_other_genes <- nuclear_genes[!nuclear_genes$gene_id %in% mrna_gene_ids & !nuclear_genes$gene_id %in% lncrna_gene_ids, ]

# Saving as csv's
write_csv(nuclear_lncRNA_genes, "results/nuclear_lncRNA_genes.csv")
write_csv(nuclear_mRNA_genes, "results/nuclear_mRNA_genes.csv")
write_csv(nuclear_other_genes, "results/nuclear_other_genes.csv")

#Breakdown of nuclear genes by gene type
nrow(nuclear_lncRNA_genes)
# 2112
nrow(nuclear_mRNA_genes)
# 2558
nrow(nuclear_other_genes)
# 1308

#How many total genes are significantly enriched in cytosol? 5978
nrow(cyto_genes)

cyto_lncRNA_genes <- cyto_genes[cyto_genes$gene_id %in% lncrna_gene_ids,]
cyto_mRNA_genes <- cyto_genes[cyto_genes$gene_id %in% mrna_gene_ids,]
cyto_other_genes <- cyto_genes[!cyto_genes$gene_id %in% mrna_gene_ids & !cyto_genes$gene_id %in% lncrna_gene_ids, ]

# Saving as csv's
write_csv(cyto_lncRNA_genes, "results/cyto_lncRNA_genes.csv")
write_csv(cyto_mRNA_genes, "results/cyto_mRNA_genes.csv")
write_csv(cyto_other_genes, "results/cyto_other_genes.csv")

#Breakdown of cyto genes by gene type
nrow(cyto_lncRNA_genes)
# 916
nrow(cyto_mRNA_genes)
# 3501
nrow(cyto_other_genes)
# 638


```


# Top enriched genes in nuclear and cytosolic fractions and gene ontology analysis
```{r nuc and cyto top genes}
#Sort dataframes and look at most different genes by log2FoldChange

# Top 10 nuclear mRNA genes
nuclear_mRNA_genes <- nuclear_mRNA_genes[order(nuclear_mRNA_genes$log2FoldChange, decreasing = FALSE),]
head(n = 10, nuclear_mRNA_genes)

# Top 10 nuclear lncRNA genes
nuclear_lncRNA_genes <- nuclear_lncRNA_genes[order(nuclear_lncRNA_genes$log2FoldChange, decreasing = FALSE),]
head(n = 10, nuclear_lncRNA_genes)

# Top 10 cyto mRNA genes (decreasing should be true because fold change is positive in this case)
cyto_mRNA_genes <- cyto_mRNA_genes[order(cyto_mRNA_genes$log2FoldChange, decreasing = TRUE),]
head(n = 10, cyto_mRNA_genes)

# Top 10 cyto lncRNA genes (decreasing should be true because fold change is positive in this case)
cyto_lncRNA_genes <- cyto_lncRNA_genes[order(cyto_lncRNA_genes$log2FoldChange, decreasing = TRUE),]
head(n = 10, cyto_lncRNA_genes)

```
Results: 
Nuclear enriched mRNA genes are associated with ECM organization and collagen fibril organization GO Biological Processes, and ECM-receptor interaction is the most enriched KEGG pathway.
Cytosolic enriched mRNA genes are associated with translation pathways and the RNA-binding GO molecular function.


# Reservoirs
```{r Reservoir analysis}

# define reservoirs as genes with >200 dbps bound and <0.01 tpm
promoter_features_df$reservoir <- 
  as.numeric(promoter_features_df$number_of_dbp > 200 & 
               promoter_features_df$tpm_homo_sapiens_hepg2 < 0.01)

# Table $reservoir to see how many 
table(promoter_features_df$reservoir)

# So there are 199 genes with superbinding promoters and that are not expressed in HepG2 cells.

# Create df with reservoir genes and check there is 199 with nrow()
reservoir_df <- promoter_features_df %>% 
  filter(promoter_features_df$reservoir == 1)
nrow(reservoir_df)

# How many reservoir promoters are lncRNA vs mRNA?
table(reservoir_df$gene_type)

# Save reservoir dataframe
write_csv(reservoir_df, "results/reservoir_genes.csv")

```
Results: 199 reservoir promoters with over 200 DBPS bound but no expression. 114 of these are lncRNA genes while the other 85 are protein coding mRNA genes.


```







